scripts/install_android.sh

```diff
@@ -7,35 +7,23 @@ dir=$(dirname "${0}")
 
 # parseJSON.js
 appID=$(${dir}/parseJSON.js $project_path andAppID)
+# If App ID doesn't exist, silently fail. Error message will show on the component.
+if [ -z "$appID" ]; then
+  exit 0;
+fi
 
 # AdMob Dependencies
 yarn add react-native-admob@^2.0.0-beta.6
 
 # build.gradle
 cd android/app
 
-# sed -i.bak '
```

Comment:
Why are these getting removed?

Reply:
They were commented out and aren't used, so I was just going to delete them.